lisa/util/shell.py

```diff
@@ -663,7 +663,8 @@ def mkdir(
             parents: make parent directories as needed
             exist_ok: return with no error if target already present
         """
-        assert isinstance(path, Path), f"actual: {type(path)}"
+        path = Path(path)
```

Comment:
It's worth to be a small, separated PR. Let's get small and isolated changes in faster.

Reply:
sure